csrc/scheduler/normalization_utils.cpp

```diff
@@ -1630,13 +1621,28 @@ void schedulePersistentKernel(
       reduction_scheduler_utils::getCachedTvsToUnrollOrVectorize(
           reference_tv, is_vectorize, cached_inputs, cached_outputs);
 
+  std::unordered_set<TensorView*> unroll_vectorizable_cached_and_cast_tvs;
+  if (rparams->vectorize_casts) {
+    for (auto tv : fusion->allTvs()) {
+      if (auto uop = dynamic_cast<UnaryOp*>(tv->definition())) {
+        if (uop->getUnaryOpType() == UnaryOpType::Cast) {
+          unroll_vectorizable_cached_and_cast_tvs.emplace(tv);
```

Comment:
I think @naoyam suggested this change https://github.com/NVIDIA/Fuser/pull/4831/files to disable it for non sub 8 bit here.